src/components/ReportPage/index.jsx

```diff
@@ -51,17 +57,30 @@ class ReportPage extends PureComponent {
     }
   }
 
-  reportData = async () => {
+  reportData = async (currentPage, limit) => {
     const { isLoadingReports } = this.state;
     this.setState({
       isLoadingReports: !isLoadingReports,
     });
-    const url = 'https://api-staging-esa.andela.com/api/v1/automations';
-    const data = await axios
+    const { REACT_APP_BACKEND_URL } = process.env;
+    const url = `${REACT_APP_BACKEND_URL}/api/v1/automations?page=${currentPage}&limit=${limit}`;
+    const reportData = await axios
       .get(url)
-      .then(response => response.data.data)
+      .then(response => response.data)
       .catch(error => error.response);
-    return data;
+
+    const offset = reportData.data && limit * (parseInt(reportData.pagination.currentPage, 10) - 1);
```

Comment:
What do you think about placing this block of code inside the `.then`? The reason I am asking this is for error handling purposes. At the moment when an error occurs, the app will crash because the variable `reportData` will not contain the property data we want.

Reply:
Makes sense, noted